### ISSUE TITLE
How to add an item to the list

### ISSUE THREAD

**Issue by jospoortvliet** [nextcloud/providers#21]
## Step 1
Edit the readme and add yourself in the correct way (copy and adjust another provider; make sure you add yourself in alphabetical order on name). [Use this link](https://github.com/nextcloud/providers/edit/master/README.md ) or click the edit icon on top of the readme.
**You need to be logged in for this to work!**
## Step 2.
Create a pull request:
![Screenshot_20200520_083527](https://user-images.githubusercontent.com/551757/82413154-2a50a200-9a75-11ea-821b-b88eff961264.png)
You can enter a description and see the difference:
![Screenshot_20200520_085732](https://user-images.githubusercontent.com/551757/82414847-f32fc000-9a77-11ea-9c50-3f2c59f9d354.png)
## Step 3.
Add your logo. Now you go and move into the 'branch' you created. see where my mouse cursor is:
![Screenshot_20200520_085818](https://user-images.githubusercontent.com/551757/82414963-207c6e00-9a78-11ea-9d7a-6d4434c2202c.png)
You navigate into the logos folder:
![Screenshot_20200520_085916](https://user-images.githubusercontent.com/551757/82415029-35590180-9a78-11ea-9762-c2a14aae28c0.png)
And there you upload the new file (make sure it has the right size!)
![Screenshot_20200520_085943](https://user-images.githubusercontent.com/551757/82415089-486bd180-9a78-11ea-9c72-78ece136e685.png)
You commit this change (the file addition) to the branch of the pull request.
![Screenshot_20200520_090036](https://user-images.githubusercontent.com/551757/82415169-6b968100-9a78-11ea-99aa-a7e7308f2474.png)
## Step 4
Now you have a pull request with the image and the readme. Check if it looks good below, then click the number to go to your PR - it should be good for somebody to review and merge.
![Screenshot_20200520_091755](https://user-images.githubusercontent.com/551757/82416740-d8127f80-9a7a-11ea-9f56-352cac225d8b.png)
![Screenshot_20200520_091846](https://user-images.githubusercontent.com/551757/82416781-ea8cb900-9a7a-11ea-972f-420975d265b0.png)

Good luck!

**Comment by espressoelf** [comment]
> Community maintained list of Nextcloud providers
but 
> add yourself 

So only **providers** are allowed to add **themselves**? Or are people allowed to add providers they know, too?

**Comment by jospoortvliet** [comment]
> > Community maintained list of Nextcloud providers
> 
> but
> 
> > add yourself
> 
> So only **providers** are allowed to add **themselves**? Or are people allowed to add providers they know, too?

Good point, yeah, I think anyone can add providers. Will update the text.

**Comment by zaclys** [comment]
Hi @jospoortvliet,

we added our request yesterday... We just hope we did it the right way : https://github.com/zaclys/providers

Best regards,
The Zaclys Team (FR)

**Comment by jospoortvliet** [comment]
> Hi @jospoortvliet,
> 
> we added our request yesterday... We just hope we did it the right way : https://github.com/zaclys/providers
> 
> Best regards,
> The Zaclys Team (FR)

Hum, something went wrong, as there is no difference between your branch and ours, as in, the text and image are not there. So you didn't upload the logo or modify the readme.md.

You can do that in your own branch at https://github.com/zaclys/providers and then make a pull request (this option is just below the top of the page).

So, in simple steps:
1. fork the repo (you did, to https://github.com/zaclys/providers)
2. add the logo image and edit readme.md to add yourself
3. check if it looks good in your repo (https://github.com/zaclys/providers in your case)
4. once you are happy, choose "create a pull request":
![Screenshot_20210208_135819](https://user-images.githubusercontent.com/551757/107222844-b9438700-6a15-11eb-873a-e0cc91890c38.png)
5. go through the steps and submit the pull request.

**Comment by zaclys** [comment]
Hi @jospoortvliet ,

oh sorry, first time for me using github... and did not smash that pull request button.

I hope now it's ok : https://github.com/nextcloud/providers/pull/56/commits/03c3bf4ede04130e8d2a504c06b1f750a14dc8b8?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

Thanks a lot for your kind support and have a very nice day Jos !
The Zaclys Team (FR)

**Comment by zaclys** [comment]
Hi @jospoortvliet , 
we hope now our PR is ready to be approved.
Best regards,
The Zaclys Team (FR)
